compiler/rustc_session/src/config.rs

```diff
@@ -1273,7 +1273,7 @@ fn default_configuration(sess: &Session) -> Cfg {
     }
     ret.insert((sym::target_arch, Some(Symbol::intern(arch))));
     ret.insert((sym::target_endian, Some(Symbol::intern(end.as_str()))));
-    ret.insert((sym::target_pointer_width, Some(Symbol::intern(&wordsz))));
+    ret.insert((sym::target_pointer_width, Some(sym::integer(wordsz))));
```

Comment:
This doesn't have any perf improvement (only 0..9 are pre-interned) This code now does have an extra check for wordsz < 10, but given that this only runs once per compiler session, it shouldn't really matter.